nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPWorker.java

```diff
@@ -47,7 +46,7 @@
      * @param snmp instance of {@link Snmp}
      * @param target instance of {@link AbstractTarget}
      */
-    public SNMPWorker(Snmp snmp, AbstractTarget target) {
+    protected SNMPWorker(Snmp snmp, AbstractTarget target) {
```

Comment:
totally agree with you on this case! I was guilty here of seeing only a limited section of code in my code review tool and made a bad assumption. Totally agree that while protected and public here are mostly equivalent, protected is a better choice.